src/test/resource/youtubeHTML5Video.feature

```diff
@@ -32,14 +30,23 @@ Feature: HTML5 Video Player functionality on Youtube
 
   @Youtube
   Scenario: Test exiting fullscreen
-    Given video is fullscreen
+    Given user presses the fullscreen button
+    And video is fullscreen
     When user presses the fullscreen button
```

Comment:
because you re-use "user pressed x button" you should parameterized it and pass in button names as arguments.